arch/arc/Kconfig

```diff
@@ -327,6 +327,43 @@ config ARC_NORMAL_FIRMWARE
 	  resources of the ARC processors, and, therefore, it shall avoid
 	  accessing them.
 
+config ARC_DSP_COMPLEX
```

Comment:
changed to CONFIG_ARC_DSP_BFLY_SHARING, it is not a configured hardware and it can be configured in `prj.conf` file, just like [dsp test](https://github.com/foss-for-synopsys-dwc-arc-processors/zephyr/blob/c9f9fb4dc9fc994fe419ee962573256bcc790681/tests/kernel/dsp_sharing/generic/prj.conf#:~:text=CONFIG_ARC_DSP_BFLY_SHARING%3Dy).